internal/api/client/handlers/auth/handlers.go

```diff
@@ -0,0 +1,61 @@
+package auth
+
+import (
+	"context"
+	"google.golang.org/grpc"
+	v1_pd "goph-keeper/internal/proto/v1"
+	"log/slog"
+)
+
+type service interface {
```

Comment:
Можно попробовать использовать нотацию, которую рекомендуют сами разработчики Go для интерфейсов из одного метода - имя метода + er.
```suggestion
type tokenSaver interface {
```